### DISCUSSION TITLE
Hybrid workshop with examples in PK-Sim® and MoBi® (4-5 November 2023)

### DISCUSSION THREAD

**Discussion by nicospinu** [Open-Systems-Pharmacology/Forum#1640] [Announcement]
Dear OSP Community, 

We are delighted to share the news of an upcoming hybrid workshop that we are happy to invite you to. 

This **two-day event** is packed with lectures and practical assignments on how to use PK-Sim® and MoBi® with **small and large molecules**. Included are topics such as drug-drug interaction (DDI), age-dependent PK in pediatrics, and protein/antibody PBPK modeling for antibody-drug-conjugates (ADCs) for the treatment of tumor growth accounting for target-mediated drug disposition (TMDD). 

The workshop will be given by our CEO and Lead Scientist, Dr. Stephan Schaller, together with our Principal Scientist, Dr. Wilbert De Witte, as a pre-meeting workshop before the upcoming ACoP14 conference. 

We offer free places for online attendance and have a limited number of free and discounted places for the in-person event. 

Book your ticket at [Workshop (ACoP14 2023) – esqLABS GmbH](https://esqlabs.shop/products/workshop-acop-2023). 

Please reach out to us for any further queries! [training@esqlabs.com](mailto:training@esqlabs.com).  

Best wishes, 
Nicoleta 
Scientist & Training Platform Lead @esqLABS

**Comment by StephanSchaller** [comment]
The ADCs example for treating tumor growth is a great use case for leveraging mechanistic PK modeling for a QSP application for a so-called PB-QSP or PBPK-QSP modeling approach.

Looking forward to this!
